docs/evaluation/examples.rst

```diff
@@ -300,6 +303,11 @@ For now we want to make sure we complete our analysis in
 energy units::
 
    >>> pha.set_analysis('energy')
+   >>> pha.notice(0.3, 7)
```

Comment:
Why did you add these lines? You need to update `docs/_examples/evaluation/examples/pharesponses.py` to reflect this (I use these scripts to make it easy to regenerate the text and figures) and then update the output below where it's changed. Or just remove these lines.

Reply:
I don't remember the PR number, but I had to rebase this because the docs where re-organized by @DougBurke in some other PR after I opened this. So, I'll go with "remove".